lib/ffmpeg/movie.rb

```diff
@@ -13,13 +13,15 @@ class Movie
 
     UNSUPPORTED_CODEC_PATTERN = /^Unsupported codec with id (\d+) for input stream (\d+)$/
 
-    def initialize(path)
+    def initialize(path, skip_head_request = true)
```

Comment:
```suggestion
    def initialize(path, skip_head_request: true)
```

Could we please use named arguments here. So, it's more clear what is this boolean argument we are passing down.

Reply:
sure, we can.